src/client/core-client.ts

```diff
@@ -63,11 +67,14 @@ interface StopServiceReply {
 }
 
 interface DeployServiceRequest {
-    service: Service
+    url: string
+    chunk: any
 }
 
 interface DeployServiceReply {
     serviceID: string
+    validationError: string
+    status: any
```

Comment:
this is not correctly defined

Reply:
I will fix the part that it should be optional, for the any, I don't actually know exactly what it will be so we can always add an issue to have better TS support but this PR is not about that so I dont think we should do that here